### PR TITLE
Remove QDebug Added Loging Box

### PR DESCRIPTION
**Pull request by sithlord48**
Reoved qdebug and added log box. 
added closeConnection() for serialLayer

### CODE REVIEW

**Comment by sithlord48** [comment]
I also added a wipe of the command to be sent when send is pressed.

File: src/mainwindow.h [hunk 1]
```diff
@@ -23,5 +23,6 @@ public slots:
     Ui::MainWindow *ui;
     void locateSerialPort();
     QStringList serialPortList;
-	QTimer _timer;
+	void click();
```
Comment (patrickelectric) [inline src/mainwindow.h]: Indentation

File: src/mainwindow.cpp [hunk 2]
```diff
@@ -46,55 +51,46 @@ void MainWindow::locateSerialPort()
...
+        ser->closeConnection();
+        disconnect(_timer,SIGNAL(timeout()),this,SLOT(checkCommand()));
+        ui->logTE->appendPlainText(QString(tr("Disconnected")));
 
-	ui->logTE->appendPlainText(QString(tr("Disconnected")));
+    } else if(btn == ui->sendPB){
+
+        QByteArray comm = ui->commandLE->text().toLocal8Bit();
+        ser->pushCommand(comm);
+		ui->commandLE->clear();
```
Comment (patrickelectric) [inline src/mainwindow.cpp]: Indentation

File: src/mainwindow.cpp [hunk 3]
```diff
@@ -46,55 +51,46 @@ void MainWindow::locateSerialPort()
...
 
-	connect(_timer,SIGNAL(timeout()),this,SLOT(checkCommand()));
-	//logging
-	ui->logTE->appendPlainText(QString(tr("Connected to %1").arg(port)));
+    if(btn == ui->connectPB ){
 
+        QString port = ui->serialPortCB->currentText();
+        uint baud = ui->baudRateLE->text().toUInt();
+        ser = new SerialLayer(port, baud);
+		_timer->setInterval(1000);
```
Comment (patrickelectric) [inline src/mainwindow.cpp]: Indentation

**Comment by patrickelectric** [comment]
@sithlord48 , Please use spaces not tabs and check your indentation.

**Pull request closed without merging**